framework/src/application.ts

```diff
@@ -304,6 +305,7 @@ export class Application {
 					.then(() => {
 						this.logger.debug(this._controller.getEvents(), 'Application listening to events');
 						this.logger.debug(this._controller.getEndpoints(), 'Application ready for actions');
+						this.logger.info(this._controller.getEndpoints(), 'Application ready for actions');
```

Comment:
```suggestion
						this.logger.info(this._controller.getEndpoints(), 'Application ready for endpoints');
```
also we can remove the `debug` level above